Sources/SwinjectStoryboard.swift

```diff
@@ -53,14 +43,30 @@ public class SwinjectStoryboard: _SwinjectStoryboardBase, SwinjectStoryboardProt
     /// - Parameters:
     ///   - name:      The name of the storyboard resource file without the filename extension.
     ///   - bundle:    The bundle containing the storyboard file and its resources. Specify nil to use the main bundle.
+    ///
+    /// - Note:
+    ///                The shared singleton container `SwinjectStoryboard.defaultContainer` is used as the container.
+    ///
+    /// - Returns: The new instance of `SwinjectStoryboard`.
+    @objc public class func create(
+        name: String,
+        bundle storyboardBundleOrNil: Bundle?) -> SwinjectStoryboard {
+        return SwinjectStoryboard.create(name: name, bundle: storyboardBundleOrNil,
+                                         container: SwinjectStoryboard.defaultContainer)
+    }
```

Comment:
This needed to be done because ObjC doesn't support Swift's default parameters (`container: Resolver = SwinjectStoryboard.defaultContainer`). These two methods are functionally equivalent to the single old method, so the API hasn't actually changed.